transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClientImpl.kt

```diff
@@ -759,7 +759,7 @@ internal class MessagingClientImpl(
 
                     is StructuredMessage -> {
                         decoded.body.run {
-                            val message = this.toMessage()
+                            val message = this.toMessage(decoded.tracingId)
```

Comment:
I see. So Shyrka put tracingId on the top level of the StructuredMessage object and not nested inside its body. Make sense.